File: permission/src/main/java/fudi/freddy/permission/PermisionInterface.java
package fudi.freddy.permission;

import java.util.ArrayList;

/**
 *
 * @author Romell Domínguez
 * @version 1.0.a 16/07/2018
 * @since 1.0
 */

public interface PermisionInterface {
    void onRefuse(ArrayList<String> RefusePermissions);
    void onFinally();
}

File: fudi-permission-kotlin/src/main/java/com/romellfudi/permission/PermisionInterface.kt
package com.romellfudi.permission

import java.util.ArrayList

/**
 *
 * @author Romell Dominguez
 * @version 1.0.a 16/07/2018
 * @since 1.0
 */

interface PermisionInterface {
    fun onRefuse(refusePermissions: ArrayList<String>)
    fun onFinally()
}
